apps/next/src/pages/Approve/components/ActionDetails/evm/EvmTokenApprovals/lib/getApprovalValue.ts

```diff
@@ -0,0 +1,34 @@
+import { MaxUint256 } from 'ethers';
+import { TokenUnit } from '@avalabs/core-utils-sdk';
+import { TokenApproval, TokenType } from '@avalabs/vm-module-types';
+
+export const getApprovalValue = (
+  approval: TokenApproval,
+  getTokenPrice: (symbolOrAddress: string) => number | undefined,
+) => {
+  if (!approval.value) {
+    return null;
+  }
+
+  const isNFT = approval.token.type !== TokenType.ERC20;
```

Comment:
This can break if new token types are introduced.
```suggestion
  const isNFT = approval.token.type === TokenType.ERC721 || approval.token.type === TokenType.ERC1155;
```